cypress/e2e/mainPage.cy.ts

```diff
@@ -0,0 +1,50 @@
+/// <reference types="cypress" />
+export type TestUser = {
+  name: string;
+  abbreviatedName: string;
+  email: string;
+  sessionToken: string;
+};
+
+describe("Checks main page content", () => {
+  beforeEach(() => {
+    cy.visit("/");
+    cy.dataCy("welcomeHeader").as("welcomeHeader");
+    cy.dataCy("welcomeText").as("welcomeText");
+  });
+
+  it("Checks if favicon is available", () => {
+    cy.request("/favicon.ico").then((response) => {
+      expect(response.status).to.eq(200); // Assert that the file exists
+    });
+  });
```

Comment:
unnecessary test in general, because this will slow down the test execution and has little to no value